iamctl/tests/keywordUtils_test.go

```diff
@@ -234,7 +234,7 @@ func TestGetKeywordLocations(t *testing.T) {
 		"claimMappings.[localClaim.claimUri=http://wso2.org/claims/identity/accountLocked].remoteClaim.claimUri",
 	}
 
-	result := utils.GetKeywordLocations(fileData, []string{}, keywordMapping)
+	result := utils.GetKeywordLocations(fileData, []string{}, keywordMapping, "test")
```

Comment:
Shall we add the resource type as APPLICATIONS here and add a separate test case with resource type CLAIMS?

Reply:
Changed the "test" strings to an actual resource type.